helm/dev-values.yaml

```diff
@@ -1,6 +1,6 @@
 environment: development
 container:
-  redisname: redis-redis-ha.mine-support
+  redisname: redis-redis-ha
```

Comment:
Does this change relate to the auth feature? Happy for it go through, just not sure how it relates.

Reply:
the dev install for Redis wasn't consistent with how it is deployed to the cluster. 
I've updated the README so Redis is installed into default rather than mine-support, so it no longer needs the '.mine-support' suffix.

In fact, the whole line can be removed, as it is the same setting as in the values file.
Good spot, I'll remove the line.